dev/com.ibm.ws.microprofile.config.1.2_fat/bnd.bnd

```diff
@@ -0,0 +1,33 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+	fat/src,\
+	test-applications/converterApp/src,\
+	test-applications/converterApp/resources
+
+javac.source: 1.8
+javac.target: 1.8
+	
+fat.project: true
+
+-buildpath: \
+	com.ibm.websphere.javaee.ejb.3.2;version=latest,\
+	com.ibm.websphere.javaee.servlet.4.0;version=latest,\
+	com.ibm.websphere.org.eclipse.microprofile.config.1.2;version=latest,\
+	com.ibm.websphere.javaee.annotation.1.3;version=latest,\
+	com.ibm.ws.security;version=latest,\
```

Comment:
these deps can probably be removed from the buildpath and will make the FAT build and run faster locally:
```com.ibm.ws.security;version=latest,\
com.ibm.websphere.security;version=latest,\```